.github/workflows/cicd.yaml

```diff
@@ -12,7 +12,7 @@ on:
 env:
   POETRY_VERSION: 1.1.13
   PSR_VERSION: 7.28.1
-  CACHE_NUMBER: 4
+  CACHE_NUMBER: 5
```

Comment:
Out of interest, could I ask what can be volatile, conditional on `poetry.lock`s being consistent?

Reply:
Me having a beta release locally. There's going to be some breaking changes when Poetry 1.2 releases for the look file, so I was testing them out. Forgot that I always use latest stable (so v1.1.13) for poetry on the build, and it took me a few checks to find the error. Reverting to 1.1.13 locally actually solved the problem, but at that point I already pushed these commits.

Reply:
Ah right, yes. Thank you for the generous explanation!